pkg/platform/abstract/platform.go

```diff
@@ -27,6 +27,10 @@ import (
 	"github.com/nuclio/logger"
 )
 
+const (
+	s3EntryType = "s3"
+)
```

Comment:
Duplication of the consts in `build` package. Capitalize them there to have them exportable and used from this package